browser/aframeComponents/wearable-skin.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Component for selecting your character's skin in game.
+ */
+
+import AFRAME from 'aframe';
+const THREE = window.THREE;
```

Comment:
yeah, the line looks insane! I must have been trying to quiet the linter. Fixing now.